enterprise-beans/4.0/apidocs/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,4 @@
+Manifest-Version: 1.0
```

Comment:
The apidocs directory should not have a META-INF directory (part of the checklist).  Please remove.

Reply:
Done